src/commonMain/kotlin/data/gems/MysticalSkyfireDiamond.kt

```diff
@@ -0,0 +1,60 @@
+package data.gems
+
+import character.Ability
+import character.Buff
+import character.Proc
+import character.Stats
+import data.model.*
+import sim.Event
+import sim.SimParticipant
+import kotlin.js.JsExport
+
+@JsExport
+class MysticalSkyfireDiamond : Gem(25893, "Mystical Skyfire Diamond", "inv_misc_gem_diamond_07.jpg", null, Color.META, Quality.META) {
+
+    val hasteBuff = object : Buff() {
+        override val name: String = "Mystical Skyfire Diamond"
+        override val durationMs: Int = 10000
```

Comment:
```suggestion
        override val durationMs: Int = 6000
```
AFAICT from internet DBs, this one should be 6s

Reply:
Might have to get someone to test this on the beta, few things I found on it

- In the 2.2 patch notes it says it was increased to 10 seconds 
- wowwiki there is mention of an undocumented change in 2.4.0 that made it a 4 second buff with a 45 sec ID, and was no longer consumed by the next spell cast (even though the previous iteration doesn't mention consumed on next cast).
- wow.tools lists it as having a 15% proc chance and 35 second ICD
- the spell that gets triggered for the haste buff is called 'focus' (ID:18803) and has a 4 second duration, even though the passive spell aura (ID: 32837) on the gem states 6 seconds (I assume it just was never updated on blizz's side)

I think it might actually be a 4 second duration with a 15% proc chance and 35 second ICD

Reply:
@Requielx Makes sense - going with 4s buff/15%/35s ICD sounds good to me.  If beta is different, easy to update.